docs/quick-start.md

```diff
@@ -24,44 +24,48 @@ doc-topic: [quick-start]
 
 The Kubewarden stack comprises:
 
-- Some [[< cluster-admission-policy >]] resources: this is how policies are defined for Kubernetes clusters
+- Some [[< cluster-admission-policy >]] resources: this is how policies are
+defined for Kubernetes clusters.
 
-- Some [[< policy-server >]] resources: representing a deployment of a Kubewarden
-`PolicyServer`. Your administrator's policies are loaded and evaluated by the Kubewarden
-`PolicyServer`
+- Some [[< policy-server >]] resources: representing a deployment of a
+Kubewarden `PolicyServer`. The Kubewarden `PolicyServer` loads and evaluates
+your administrator's policies.
 
-- Some [[< admission-policy >]] resources: policies for a defined namespace
+- Some [[< admission-policy >]] resources: policies for a defined namespace.
 
-- A deployment of a `kubewarden-controller`: this controller monitors the
-[[< cluster-admission-policy >]] resources and interacts with the Kubewarden
-[[< policy-server >]] components.
+- A deployment of a `kubewarden-controller`: this controller monitors the [[<
+cluster-admission-policy >]] resources and interacts with the Kubewarden [[<
```

Comment:
the whitespace changes on these two lines here broke the [variable rendering syntax](https://github.com/kubewarden/docs/blob/a1ac1a8b84727bbe7c1f9a2b6956dc3a83688f98/js-lib/docusaurus-variables.js#L11):
![Screenshot 2025-03-02 at 8 50 14 PM](https://github.com/user-attachments/assets/1057a8df-8de6-4d78-851e-ecc1030172c2)

I have fixed this in #549